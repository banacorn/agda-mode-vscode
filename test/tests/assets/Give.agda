module Give where

data ℕ : Set where
  zero : ℕ
  suc : ℕ → ℕ

_+_ : ℕ → ℕ → ℕ
zero + y = {!   !}
suc x + y = {!   !}