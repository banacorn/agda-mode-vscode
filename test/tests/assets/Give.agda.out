module Give where

data ℕ : Set where
  zero : ℕ
  suc : ℕ → ℕ

_+_ : ℕ → ℕ → ℕ
zero + y = y
suc x + y = {!   !}